uisdk/src/main/java/com/karhoo/uisdk/screen/booking/quotes/filterview/MultiSelectCheckboxFilterView.kt

```diff
@@ -22,19 +22,20 @@ class MultiSelectCheckboxFilterView  @JvmOverloads constructor(context: Context,
             value?.invoke()
         }
 
-    var choices: HashMap<String, String> = HashMap()
+    var choices = ArrayList<MultiSelectData>()
         set(value) {
             field = value
-            for((key, mapValue) in choices){
+            value.forEach { data ->
                 val box = CheckBox(ContextThemeWrapper(context, R.style.KhFilterViewCheckBox))
-                box.text = key
-                if(filter?.selectedTypes?.contains(mapValue) == true)
+                box.text = data.text
+                if(filter?.selectedTypes?.contains(data) == true)
                     box.isChecked = true
```

Comment:
Can we replace this `if` with `box.isChecked = filter?.selectedTypes?.contains(data)`